docs/explanation/remote-build.rst

```diff
@@ -0,0 +1,172 @@
+Remote build
+============
+
+Remote build is a feature in snapcraft that offloads the build process to
+`Launchpad`_'s `build farm`_ and enables developers to build snaps for
+different architectures.
+
+Architectures supported by Launchpad can be found
+:ref:`here<supported-architectures>`.
+
+Open vs closed source
+---------------------
+
+By default, prospective snaps must be open source because the build will be
+publicly available.
+
+Developers are reminded of this by confirming that their project will be
+publicly available when starting a remote build. This prompt can be
+automatically agreed to by passing ``--launchpad-accept-public-upload``.
+
+Closed-source projects can be built using the remote builder. This requires
+the user to create a private Launchpad project and pass the project with the
+``--project <project-name>`` command line argument.
+
+Git repository
+--------------
+
+Projects must be in the top level of a git repository because snapcraft uses
+a git-based workflow to upload projects to Launchpad.
+
+Shallowly cloned repositories are not supported (i.e. ``git clone --depth 1``)
```

Comment:
```suggestion
Shallowly cloned repositories are not supported (e.g. ``git clone --depth 1``)
```

Because this is an example right? That is, ``git clone --depth 2`` also wouldn't work